lua/pac3/editor/client/wear.lua

```diff
@@ -258,9 +258,18 @@ do
 		elseif pace.IsActive() then
 			pac.Message("not wearing autoload outfit, editor is open")
 		else
-			pac.Message("Wearing autoload...")
-			pace.LoadParts("autoload")
-			pace.WearParts()
+			local autoload_state = "autoload"
+			autoload_state = hook.Run("PAC3Autoload", autoload_state)
```

Comment:
This could just be `local autoload_state = hook.Run("PAC3Autoload", autoload_state)`